nextjs/about/pages/[...slug].js

```diff
@@ -21,16 +21,33 @@ const CatchAll = ({ Page, ...props }) => {
     </>
   );
 };
+
 CatchAll.getInitialProps = async ({ err, req, res, AppTree, ...props }) => {
   const pageName = `./${props.query.slug}`;
+
   if (process.browser) {
+    let container;
+    console.log("page", `${props.query.slug}`);
+    switch (`${props.query.slug}`) {
+      case "info":
```

Comment:
primitive router mapping needs an npm plugin or abstraction to be created that's more robust